src/accel/CylMapMagneticField.cc

```diff
@@ -55,31 +55,30 @@ inline void cartesian_to_cylindrical(Array<G4double, 3> const& cart,
  * it will retrieve the G4FieldManager's field to sample it.
  */
 CylMapFieldParams::Input
-MakeCylMapFieldInput(std::vector<real_type> const& r_grid,
-                     std::vector<real_type> const& phi_values,
-                     std::vector<real_type> const& z_grid)
+MakeCylMapFieldInput(std::vector<float> const& r_grid,
+                     std::vector<float> const& phi_values,
+                     std::vector<float> const& z_grid)
```

Comment:
The geant4 interface functions should all take `double` I think. We should convert internally to real_type.